lisa/operating_system.py

```diff
@@ -16,6 +18,42 @@
 
 _get_init_logger = partial(get_logger, name="os")
 
+LSB_RELEASE_PATTERN = re.compile(r"^(?P<name>.*):(\s+)(?P<value>.*?)?$", re.MULTILINE)
```

Comment:
Those patterns are used in Posix and its subclasses only, so put them into the `Posix` class or lower level subclass. It doesn't need to put in the module level.

Reply:
ok moved. But in that case, we should move the __get_detect_string() and all the pattern matching it uses to Posix too?
I don't see any of those things being used anywhere apart from Posix and its sub-classes.?